workspaces/applications/Makefile

```diff
@@ -0,0 +1,25 @@
+# Applications Makefile
+#
+# AUTHORS
+#
+# The Veracruz Development Team.
+#
+# COPYRIGHT AND LICENSING
+#
+# See the `LICENSING.markdown` file in the Veracruz root directory for
+# licensing and copyright information.
+
+.PHONY: all doc clean fmt
+
+all:
+	rustup target add wasm32-wasi
```

Comment:
I like this. Doing it in the `dockerfile` always created problems[

Reply:
To be clear my Docker update PR does install it by default (https://github.com/veracruz-project/veracruz-docker-image/pull/29/files#diff-b093d34eec92adf6e0cc6e445edb7147ea45d25fb0c98f2f84a60f5aae6f34f7R103), but if there is a change in rust-toolchain, this ensure it is updated, the rest of the time it's idempotent.